plugins/repository-s3/src/test/java/org/opensearch/repositories/s3/S3BlobContainerRetriesTests.java

```diff
@@ -162,12 +164,17 @@ protected BlobContainer createBlobContainer(
         ) {
             @Override
             public InputStream readBlob(String blobName) throws IOException {
-                return new AssertingInputStream(super.readBlob(blobName), blobName);
+                return new AssertingInputStream(SocketAccess.doPrivilegedIOException(() -> super.readBlob(blobName)), blobName);
             }
 
             @Override
             public InputStream readBlob(String blobName, long position, long length) throws IOException {
-                return new AssertingInputStream(super.readBlob(blobName, position, length), blobName, position, length);
+                return new AssertingInputStream(
+                    SocketAccess.doPrivilegedIOException(() -> super.readBlob(blobName, position, length)),
```

Comment:
And here?

Reply:
Same as above, thanks for calling out @reta.